docker/release/cudaq.nvcf.Dockerfile

```diff
@@ -0,0 +1,26 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+# This file is used to build CUDA Quantum NVCF service container.
+#
+# Usage:
+# Must be built from the repo root with:
+#   DOCKER_BUILDKIT=1 docker build -f docker/release/cudaq.nvcf.Dockerfile . --output out
+
+# Base image is CUDA Quantum image 
+ARG base_image=nvcr.io/nvidia/nightly/cuda-quantum:latest
+FROM $base_image as nvcf_image
+
+USER root
+
+ENV CUDAQ_LOG_LEVEL=info
+ADD ./scripts/launch_qpud.sh /launch_qpud.sh
```

Comment:
Seeing as this is just one command, it seems unnecessary to add a separate script.
Why not just 
```
RUN echo 'mpiexec --allow-run-as-root -np $(nvidia-smi --list-gpus | wc -l) cudaq-qpud --type nvcf' > launch.sh
```
and
```
ENTRYPOINT ["bash", "-l", "launch.sh"]
```

Reply:
Good point. I've updated the file as suggested (without `--allow-run-as-root`) and removed the `launch_qpud.sh` file.